.storybook/stories/CorporateCampaign.stories.js

```diff
@@ -0,0 +1,58 @@
+import Vue from 'vue';
+import StoryRouter from 'storybook-vue-router';
+import { withKnobs, text, object } from '@storybook/addon-knobs';
```

Comment:
Minor, these aren't needed since you're using the `args`, and `argTypes` syntax

Reply:
Good to know...

I need to figure out how to segment those automatic options/args so that only the applicable ones show up in each story...